src/Ast/PhpParam.php

```diff
@@ -41,8 +41,15 @@ public function __construct(?ResolvedType $type, Variable $var, $default)
 
     public function toCode(): string
     {
-        $type = is_null($this->type) ? '' : static::toPhp($this->type) . ' ';
-        $default = is_null($this->default) ? '' : ' = ' . static::toPhp($this->default);
+        $type = null;
+        if ($this->type) {
+            $type = substr($this->type->type->name, -2) === '[]'
```

Comment:
I'll cut this out for now, as it is a remnant of something I refactored out anyways. It may end up being useful for the upcoming work to modify our method signatures for V2, so i'll implement something there if we need it.